core/src/main/scala/org/apache/spark/input/PortableDataStream.scala

```diff
@@ -194,5 +195,8 @@ class PortableDataStream(
   }
 
   def getPath(): String = path
+
+  @Since("2.2.0")
```

Comment:
SGTM, can you take a look at other public methods in this class and add since tag for them? or it looks weird that only one method has since tag...